<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>GO and KEGG analysis for MAPP paper</title>
  <meta name="description" content="GO and KEGG analysis for MAPP paper" />
  <meta name="generator" content="bookdown 0.29 and GitBook 2.6.7" />

  <meta property="og:title" content="GO and KEGG analysis for MAPP paper" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="GO and KEGG analysis for MAPP paper" />
  <meta name="github-repo" content="rstudio/bookdown-demo" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="GO and KEGG analysis for MAPP paper" />
  
  <meta name="twitter:description" content="GO and KEGG analysis for MAPP paper" />
  

<meta name="author" content="Ian U. Kouzel" />


<meta name="date" content="2023-02-20" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  

<link rel="next" href="go-and-kegg-common-targets-06.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<script src="libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.25/datatables.js"></script>
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.11.3/js/jquery.dataTables.min.js"></script>
<link href="libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">MAPP GO KEGG</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Prerequisites</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#libraries"><i class="fa fa-check"></i><b>1.1</b> Libraries</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#options"><i class="fa fa-check"></i><b>1.2</b> Options</a></li>
<li class="chapter" data-level="1.3" data-path="index.html"><a href="index.html#universe-set-of-genes"><i class="fa fa-check"></i><b>1.3</b> Universe set of genes</a>
<ul>
<li class="chapter" data-level="1.3.1" data-path="index.html"><a href="index.html#go-ensembl"><i class="fa fa-check"></i><b>1.3.1</b> GO (ENSEMBL)</a></li>
<li class="chapter" data-level="1.3.2" data-path="index.html"><a href="index.html#kegg-entrez"><i class="fa fa-check"></i><b>1.3.2</b> KEGG (ENTREZ)</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="index.html"><a href="index.html#render-the-book"><i class="fa fa-check"></i><b>1.4</b> Render the book</a></li>
<li class="chapter" data-level="1.5" data-path="index.html"><a href="index.html#session-info"><i class="fa fa-check"></i><b>1.5</b> Session info</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="go-and-kegg-common-targets-06.html"><a href="go-and-kegg-common-targets-06.html"><i class="fa fa-check"></i><b>2</b> GO and KEGG: common targets 06</a>
<ul>
<li class="chapter" data-level="2.1" data-path="go-and-kegg-common-targets-06.html"><a href="go-and-kegg-common-targets-06.html#load-the-data"><i class="fa fa-check"></i><b>2.1</b> Load the data</a></li>
<li class="chapter" data-level="2.2" data-path="go-and-kegg-common-targets-06.html"><a href="go-and-kegg-common-targets-06.html#gene-ontology-go"><i class="fa fa-check"></i><b>2.2</b> Gene ontology (GO)</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="go-and-kegg-common-targets-06.html"><a href="go-and-kegg-common-targets-06.html#prepare-the-gene-list"><i class="fa fa-check"></i><b>2.2.1</b> Prepare the gene list</a></li>
<li class="chapter" data-level="2.2.2" data-path="go-and-kegg-common-targets-06.html"><a href="go-and-kegg-common-targets-06.html#biological-process-bp"><i class="fa fa-check"></i><b>2.2.2</b> Biological process (BP)</a></li>
<li class="chapter" data-level="2.2.3" data-path="go-and-kegg-common-targets-06.html"><a href="go-and-kegg-common-targets-06.html#molecular-function-mf"><i class="fa fa-check"></i><b>2.2.3</b> Molecular function (MF)</a></li>
<li class="chapter" data-level="2.2.4" data-path="go-and-kegg-common-targets-06.html"><a href="go-and-kegg-common-targets-06.html#cellular-component-cc"><i class="fa fa-check"></i><b>2.2.4</b> Cellular component (CC)</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="go-and-kegg-common-targets-06.html"><a href="go-and-kegg-common-targets-06.html#kegg"><i class="fa fa-check"></i><b>2.3</b> KEGG</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="go-and-kegg-common-targets-06.html"><a href="go-and-kegg-common-targets-06.html#prepare-the-the-data"><i class="fa fa-check"></i><b>2.3.1</b> Prepare the the data</a></li>
<li class="chapter" data-level="2.3.2" data-path="go-and-kegg-common-targets-06.html"><a href="go-and-kegg-common-targets-06.html#run-enrichkegg"><i class="fa fa-check"></i><b>2.3.2</b> run enrichKEGG</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="go-and-kegg-common-targets-09.html"><a href="go-and-kegg-common-targets-09.html"><i class="fa fa-check"></i><b>3</b> GO and KEGG: common targets 09</a>
<ul>
<li class="chapter" data-level="3.1" data-path="go-and-kegg-common-targets-09.html"><a href="go-and-kegg-common-targets-09.html#load-the-data-1"><i class="fa fa-check"></i><b>3.1</b> Load the data</a></li>
<li class="chapter" data-level="3.2" data-path="go-and-kegg-common-targets-09.html"><a href="go-and-kegg-common-targets-09.html#gene-ontology-go-1"><i class="fa fa-check"></i><b>3.2</b> Gene ontology (GO)</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="go-and-kegg-common-targets-09.html"><a href="go-and-kegg-common-targets-09.html#prepare-the-gene-list-1"><i class="fa fa-check"></i><b>3.2.1</b> Prepare the gene list</a></li>
<li class="chapter" data-level="3.2.2" data-path="go-and-kegg-common-targets-09.html"><a href="go-and-kegg-common-targets-09.html#biological-process-bp-1"><i class="fa fa-check"></i><b>3.2.2</b> Biological process (BP)</a></li>
<li class="chapter" data-level="3.2.3" data-path="go-and-kegg-common-targets-09.html"><a href="go-and-kegg-common-targets-09.html#molecular-function-mf-1"><i class="fa fa-check"></i><b>3.2.3</b> Molecular function (MF)</a></li>
<li class="chapter" data-level="3.2.4" data-path="go-and-kegg-common-targets-09.html"><a href="go-and-kegg-common-targets-09.html#cellular-component-cc-1"><i class="fa fa-check"></i><b>3.2.4</b> Cellular component (CC)</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="go-and-kegg-common-targets-09.html"><a href="go-and-kegg-common-targets-09.html#kegg-1"><i class="fa fa-check"></i><b>3.3</b> KEGG</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="go-and-kegg-common-targets-09.html"><a href="go-and-kegg-common-targets-09.html#prepare-the-the-data-1"><i class="fa fa-check"></i><b>3.3.1</b> Prepare the the data</a></li>
<li class="chapter" data-level="3.3.2" data-path="go-and-kegg-common-targets-09.html"><a href="go-and-kegg-common-targets-09.html#run-enrichkegg-1"><i class="fa fa-check"></i><b>3.3.2</b> run enrichKEGG</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="go-and-kegg-ptbp1-targets-06.html"><a href="go-and-kegg-ptbp1-targets-06.html"><i class="fa fa-check"></i><b>4</b> GO and KEGG: PTBP1 targets 06</a>
<ul>
<li class="chapter" data-level="4.1" data-path="go-and-kegg-ptbp1-targets-06.html"><a href="go-and-kegg-ptbp1-targets-06.html#load-the-data-2"><i class="fa fa-check"></i><b>4.1</b> Load the data</a></li>
<li class="chapter" data-level="4.2" data-path="go-and-kegg-ptbp1-targets-06.html"><a href="go-and-kegg-ptbp1-targets-06.html#gene-ontology-go-2"><i class="fa fa-check"></i><b>4.2</b> Gene ontology (GO)</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="go-and-kegg-ptbp1-targets-06.html"><a href="go-and-kegg-ptbp1-targets-06.html#prepare-the-gene-list-2"><i class="fa fa-check"></i><b>4.2.1</b> Prepare the gene list</a></li>
<li class="chapter" data-level="4.2.2" data-path="go-and-kegg-ptbp1-targets-06.html"><a href="go-and-kegg-ptbp1-targets-06.html#biological-process-bp-2"><i class="fa fa-check"></i><b>4.2.2</b> Biological process (BP)</a></li>
<li class="chapter" data-level="4.2.3" data-path="go-and-kegg-ptbp1-targets-06.html"><a href="go-and-kegg-ptbp1-targets-06.html#molecular-function-mf-2"><i class="fa fa-check"></i><b>4.2.3</b> Molecular function (MF)</a></li>
<li class="chapter" data-level="4.2.4" data-path="go-and-kegg-ptbp1-targets-06.html"><a href="go-and-kegg-ptbp1-targets-06.html#cellular-component-cc-2"><i class="fa fa-check"></i><b>4.2.4</b> Cellular component (CC)</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="go-and-kegg-ptbp1-targets-06.html"><a href="go-and-kegg-ptbp1-targets-06.html#kegg-2"><i class="fa fa-check"></i><b>4.3</b> KEGG</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="go-and-kegg-ptbp1-targets-06.html"><a href="go-and-kegg-ptbp1-targets-06.html#prepare-the-the-data-2"><i class="fa fa-check"></i><b>4.3.1</b> Prepare the the data</a></li>
<li class="chapter" data-level="4.3.2" data-path="go-and-kegg-ptbp1-targets-06.html"><a href="go-and-kegg-ptbp1-targets-06.html#run-enrichkegg-2"><i class="fa fa-check"></i><b>4.3.2</b> run enrichKEGG</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="go-and-kegg-ptbp1-targets-09.html"><a href="go-and-kegg-ptbp1-targets-09.html"><i class="fa fa-check"></i><b>5</b> GO and KEGG: PTBP1 targets 09</a>
<ul>
<li class="chapter" data-level="5.1" data-path="go-and-kegg-ptbp1-targets-09.html"><a href="go-and-kegg-ptbp1-targets-09.html#load-the-data-3"><i class="fa fa-check"></i><b>5.1</b> Load the data</a></li>
<li class="chapter" data-level="5.2" data-path="go-and-kegg-ptbp1-targets-09.html"><a href="go-and-kegg-ptbp1-targets-09.html#gene-ontology-go-3"><i class="fa fa-check"></i><b>5.2</b> Gene ontology (GO)</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="go-and-kegg-ptbp1-targets-09.html"><a href="go-and-kegg-ptbp1-targets-09.html#prepare-the-gene-list-3"><i class="fa fa-check"></i><b>5.2.1</b> Prepare the gene list</a></li>
<li class="chapter" data-level="5.2.2" data-path="go-and-kegg-ptbp1-targets-09.html"><a href="go-and-kegg-ptbp1-targets-09.html#biological-process-bp-3"><i class="fa fa-check"></i><b>5.2.2</b> Biological process (BP)</a></li>
<li class="chapter" data-level="5.2.3" data-path="go-and-kegg-ptbp1-targets-09.html"><a href="go-and-kegg-ptbp1-targets-09.html#molecular-function-mf-3"><i class="fa fa-check"></i><b>5.2.3</b> Molecular function (MF)</a></li>
<li class="chapter" data-level="5.2.4" data-path="go-and-kegg-ptbp1-targets-09.html"><a href="go-and-kegg-ptbp1-targets-09.html#cellular-component-cc-3"><i class="fa fa-check"></i><b>5.2.4</b> Cellular component (CC)</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="go-and-kegg-ptbp1-targets-09.html"><a href="go-and-kegg-ptbp1-targets-09.html#kegg-3"><i class="fa fa-check"></i><b>5.3</b> KEGG</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="go-and-kegg-ptbp1-targets-09.html"><a href="go-and-kegg-ptbp1-targets-09.html#prepare-the-the-data-3"><i class="fa fa-check"></i><b>5.3.1</b> Prepare the the data</a></li>
<li class="chapter" data-level="5.3.2" data-path="go-and-kegg-ptbp1-targets-09.html"><a href="go-and-kegg-ptbp1-targets-09.html#run-enrichkegg-3"><i class="fa fa-check"></i><b>5.3.2</b> run enrichKEGG</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="go-and-kegg-rbfox1-targets-06.html"><a href="go-and-kegg-rbfox1-targets-06.html"><i class="fa fa-check"></i><b>6</b> GO and KEGG: RBFOX1 targets 06</a>
<ul>
<li class="chapter" data-level="6.1" data-path="go-and-kegg-rbfox1-targets-06.html"><a href="go-and-kegg-rbfox1-targets-06.html#load-the-data-4"><i class="fa fa-check"></i><b>6.1</b> Load the data</a></li>
<li class="chapter" data-level="6.2" data-path="go-and-kegg-rbfox1-targets-06.html"><a href="go-and-kegg-rbfox1-targets-06.html#gene-ontology-go-4"><i class="fa fa-check"></i><b>6.2</b> Gene ontology (GO)</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="go-and-kegg-rbfox1-targets-06.html"><a href="go-and-kegg-rbfox1-targets-06.html#prepare-the-gene-list-4"><i class="fa fa-check"></i><b>6.2.1</b> Prepare the gene list</a></li>
<li class="chapter" data-level="6.2.2" data-path="go-and-kegg-rbfox1-targets-06.html"><a href="go-and-kegg-rbfox1-targets-06.html#biological-process-bp-4"><i class="fa fa-check"></i><b>6.2.2</b> Biological process (BP)</a></li>
<li class="chapter" data-level="6.2.3" data-path="go-and-kegg-rbfox1-targets-06.html"><a href="go-and-kegg-rbfox1-targets-06.html#molecular-function-mf-4"><i class="fa fa-check"></i><b>6.2.3</b> Molecular function (MF)</a></li>
<li class="chapter" data-level="6.2.4" data-path="go-and-kegg-rbfox1-targets-06.html"><a href="go-and-kegg-rbfox1-targets-06.html#cellular-component-cc-4"><i class="fa fa-check"></i><b>6.2.4</b> Cellular component (CC)</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="go-and-kegg-rbfox1-targets-06.html"><a href="go-and-kegg-rbfox1-targets-06.html#kegg-4"><i class="fa fa-check"></i><b>6.3</b> KEGG</a>
<ul>
<li class="chapter" data-level="6.3.1" data-path="go-and-kegg-rbfox1-targets-06.html"><a href="go-and-kegg-rbfox1-targets-06.html#prepare-the-the-data-4"><i class="fa fa-check"></i><b>6.3.1</b> Prepare the the data</a></li>
<li class="chapter" data-level="6.3.2" data-path="go-and-kegg-rbfox1-targets-06.html"><a href="go-and-kegg-rbfox1-targets-06.html#run-enrichkegg-4"><i class="fa fa-check"></i><b>6.3.2</b> run enrichKEGG</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="go-and-kegg-rbfox1-targets-09.html"><a href="go-and-kegg-rbfox1-targets-09.html"><i class="fa fa-check"></i><b>7</b> GO and KEGG: RBFOX1 targets 09</a>
<ul>
<li class="chapter" data-level="7.1" data-path="go-and-kegg-rbfox1-targets-09.html"><a href="go-and-kegg-rbfox1-targets-09.html#load-the-data-5"><i class="fa fa-check"></i><b>7.1</b> Load the data</a></li>
<li class="chapter" data-level="7.2" data-path="go-and-kegg-rbfox1-targets-09.html"><a href="go-and-kegg-rbfox1-targets-09.html#gene-ontology-go-5"><i class="fa fa-check"></i><b>7.2</b> Gene ontology (GO)</a>
<ul>
<li class="chapter" data-level="7.2.1" data-path="go-and-kegg-rbfox1-targets-09.html"><a href="go-and-kegg-rbfox1-targets-09.html#prepare-the-gene-list-5"><i class="fa fa-check"></i><b>7.2.1</b> Prepare the gene list</a></li>
<li class="chapter" data-level="7.2.2" data-path="go-and-kegg-rbfox1-targets-09.html"><a href="go-and-kegg-rbfox1-targets-09.html#biological-process-bp-5"><i class="fa fa-check"></i><b>7.2.2</b> Biological process (BP)</a></li>
<li class="chapter" data-level="7.2.3" data-path="go-and-kegg-rbfox1-targets-09.html"><a href="go-and-kegg-rbfox1-targets-09.html#molecular-function-mf-5"><i class="fa fa-check"></i><b>7.2.3</b> Molecular function (MF)</a></li>
<li class="chapter" data-level="7.2.4" data-path="go-and-kegg-rbfox1-targets-09.html"><a href="go-and-kegg-rbfox1-targets-09.html#cellular-component-cc-5"><i class="fa fa-check"></i><b>7.2.4</b> Cellular component (CC)</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="go-and-kegg-rbfox1-targets-09.html"><a href="go-and-kegg-rbfox1-targets-09.html#kegg-5"><i class="fa fa-check"></i><b>7.3</b> KEGG</a>
<ul>
<li class="chapter" data-level="7.3.1" data-path="go-and-kegg-rbfox1-targets-09.html"><a href="go-and-kegg-rbfox1-targets-09.html#prepare-the-the-data-5"><i class="fa fa-check"></i><b>7.3.1</b> Prepare the the data</a></li>
<li class="chapter" data-level="7.3.2" data-path="go-and-kegg-rbfox1-targets-09.html"><a href="go-and-kegg-rbfox1-targets-09.html#run-enrichkegg-5"><i class="fa fa-check"></i><b>7.3.2</b> run enrichKEGG</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="go-and-kegg-non-targets-06.html"><a href="go-and-kegg-non-targets-06.html"><i class="fa fa-check"></i><b>8</b> GO and KEGG: non targets 06</a>
<ul>
<li class="chapter" data-level="8.1" data-path="go-and-kegg-non-targets-06.html"><a href="go-and-kegg-non-targets-06.html#load-the-data-6"><i class="fa fa-check"></i><b>8.1</b> Load the data</a></li>
<li class="chapter" data-level="8.2" data-path="go-and-kegg-non-targets-06.html"><a href="go-and-kegg-non-targets-06.html#gene-ontology-go-6"><i class="fa fa-check"></i><b>8.2</b> Gene ontology (GO)</a>
<ul>
<li class="chapter" data-level="8.2.1" data-path="go-and-kegg-non-targets-06.html"><a href="go-and-kegg-non-targets-06.html#prepare-the-gene-list-6"><i class="fa fa-check"></i><b>8.2.1</b> Prepare the gene list</a></li>
<li class="chapter" data-level="8.2.2" data-path="go-and-kegg-non-targets-06.html"><a href="go-and-kegg-non-targets-06.html#biological-process-bp-6"><i class="fa fa-check"></i><b>8.2.2</b> Biological process (BP)</a></li>
<li class="chapter" data-level="8.2.3" data-path="go-and-kegg-non-targets-06.html"><a href="go-and-kegg-non-targets-06.html#molecular-function-mf-6"><i class="fa fa-check"></i><b>8.2.3</b> Molecular function (MF)</a></li>
<li class="chapter" data-level="8.2.4" data-path="go-and-kegg-non-targets-06.html"><a href="go-and-kegg-non-targets-06.html#cellular-component-cc-6"><i class="fa fa-check"></i><b>8.2.4</b> Cellular component (CC)</a></li>
</ul></li>
<li class="chapter" data-level="8.3" data-path="go-and-kegg-non-targets-06.html"><a href="go-and-kegg-non-targets-06.html#kegg-6"><i class="fa fa-check"></i><b>8.3</b> KEGG</a>
<ul>
<li class="chapter" data-level="8.3.1" data-path="go-and-kegg-non-targets-06.html"><a href="go-and-kegg-non-targets-06.html#prepare-the-the-data-6"><i class="fa fa-check"></i><b>8.3.1</b> Prepare the the data</a></li>
<li class="chapter" data-level="8.3.2" data-path="go-and-kegg-non-targets-06.html"><a href="go-and-kegg-non-targets-06.html#run-enrichkegg-6"><i class="fa fa-check"></i><b>8.3.2</b> run enrichKEGG</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="9" data-path="go-and-kegg-non-targets-09.html"><a href="go-and-kegg-non-targets-09.html"><i class="fa fa-check"></i><b>9</b> GO and KEGG: non targets 09</a>
<ul>
<li class="chapter" data-level="9.1" data-path="go-and-kegg-non-targets-09.html"><a href="go-and-kegg-non-targets-09.html#load-the-data-7"><i class="fa fa-check"></i><b>9.1</b> Load the data</a></li>
<li class="chapter" data-level="9.2" data-path="go-and-kegg-non-targets-09.html"><a href="go-and-kegg-non-targets-09.html#gene-ontology-go-7"><i class="fa fa-check"></i><b>9.2</b> Gene ontology (GO)</a>
<ul>
<li class="chapter" data-level="9.2.1" data-path="go-and-kegg-non-targets-09.html"><a href="go-and-kegg-non-targets-09.html#prepare-the-gene-list-7"><i class="fa fa-check"></i><b>9.2.1</b> Prepare the gene list</a></li>
<li class="chapter" data-level="9.2.2" data-path="go-and-kegg-non-targets-09.html"><a href="go-and-kegg-non-targets-09.html#biological-process-bp-7"><i class="fa fa-check"></i><b>9.2.2</b> Biological process (BP)</a></li>
<li class="chapter" data-level="9.2.3" data-path="go-and-kegg-non-targets-09.html"><a href="go-and-kegg-non-targets-09.html#molecular-function-mf-7"><i class="fa fa-check"></i><b>9.2.3</b> Molecular function (MF)</a></li>
<li class="chapter" data-level="9.2.4" data-path="go-and-kegg-non-targets-09.html"><a href="go-and-kegg-non-targets-09.html#cellular-component-cc-7"><i class="fa fa-check"></i><b>9.2.4</b> Cellular component (CC)</a></li>
</ul></li>
<li class="chapter" data-level="9.3" data-path="go-and-kegg-non-targets-09.html"><a href="go-and-kegg-non-targets-09.html#kegg-7"><i class="fa fa-check"></i><b>9.3</b> KEGG</a>
<ul>
<li class="chapter" data-level="9.3.1" data-path="go-and-kegg-non-targets-09.html"><a href="go-and-kegg-non-targets-09.html#prepare-the-the-data-7"><i class="fa fa-check"></i><b>9.3.1</b> Prepare the the data</a></li>
<li class="chapter" data-level="9.3.2" data-path="go-and-kegg-non-targets-09.html"><a href="go-and-kegg-non-targets-09.html#run-enrichkegg-7"><i class="fa fa-check"></i><b>9.3.2</b> run enrichKEGG</a></li>
</ul></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">GO and KEGG analysis for MAPP paper</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="header">
<h1 class="title">GO and KEGG analysis for MAPP paper</h1>
<p class="author"><em>Ian U. Kouzel</em></p>
<p class="date"><em>2023-02-20</em></p>
</div>
<div id="prerequisites" class="section level1 hasAnchor" number="1">
<h1><span class="header-section-number">Chapter 1</span> Prerequisites<a href="index.html#prerequisites" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<div id="libraries" class="section level2 hasAnchor" number="1.1">
<h2><span class="header-section-number">1.1</span> Libraries<a href="index.html#libraries" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="index.html#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span>
<span id="cb1-2"><a href="index.html#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(clusterProfiler)</span>
<span id="cb1-3"><a href="index.html#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(pathview)</span>
<span id="cb1-4"><a href="index.html#cb1-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(enrichplot)</span>
<span id="cb1-5"><a href="index.html#cb1-5" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(org.Hs.eg.db)</span>
<span id="cb1-6"><a href="index.html#cb1-6" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(readr)</span>
<span id="cb1-7"><a href="index.html#cb1-7" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(DT)</span>
<span id="cb1-8"><a href="index.html#cb1-8" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(matrixTests)</span></code></pre></div>
</div>
<div id="options" class="section level2 hasAnchor" number="1.2">
<h2><span class="header-section-number">1.2</span> Options<a href="index.html#options" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="index.html#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="co"># These options are used for GO analysis in the downstream scripts /</span></span>
<span id="cb2-2"><a href="index.html#cb2-2" aria-hidden="true" tabindex="-1"></a><span class="co"># R markdown pages:</span></span>
<span id="cb2-3"><a href="index.html#cb2-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-4"><a href="index.html#cb2-4" aria-hidden="true" tabindex="-1"></a><span class="co"># minimal size of genes annotated by ontology term for testing</span></span>
<span id="cb2-5"><a href="index.html#cb2-5" aria-hidden="true" tabindex="-1"></a>minGSSize <span class="ot">&lt;-</span> <span class="dv">10</span> </span>
<span id="cb2-6"><a href="index.html#cb2-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-7"><a href="index.html#cb2-7" aria-hidden="true" tabindex="-1"></a><span class="co"># human DB</span></span>
<span id="cb2-8"><a href="index.html#cb2-8" aria-hidden="true" tabindex="-1"></a>organismDB <span class="ot">&lt;-</span> <span class="st">&quot;org.Hs.eg.db&quot;</span></span>
<span id="cb2-9"><a href="index.html#cb2-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-10"><a href="index.html#cb2-10" aria-hidden="true" tabindex="-1"></a><span class="co"># p-value cutoff</span></span>
<span id="cb2-11"><a href="index.html#cb2-11" aria-hidden="true" tabindex="-1"></a>pvalueCutoff <span class="ot">&lt;-</span> <span class="fl">0.1</span></span>
<span id="cb2-12"><a href="index.html#cb2-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-13"><a href="index.html#cb2-13" aria-hidden="true" tabindex="-1"></a><span class="co"># number of GOs to plot</span></span>
<span id="cb2-14"><a href="index.html#cb2-14" aria-hidden="true" tabindex="-1"></a>showCategory <span class="ot">&lt;-</span> <span class="dv">10</span> </span>
<span id="cb2-15"><a href="index.html#cb2-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-16"><a href="index.html#cb2-16" aria-hidden="true" tabindex="-1"></a><span class="co"># p-adjusted cut off for cassette exons that are different between </span></span>
<span id="cb2-17"><a href="index.html#cb2-17" aria-hidden="true" tabindex="-1"></a><span class="co"># control and cancer conditions</span></span>
<span id="cb2-18"><a href="index.html#cb2-18" aria-hidden="true" tabindex="-1"></a>genes_padj_cutoff <span class="ot">&lt;-</span> <span class="fl">0.1</span></span></code></pre></div>
</div>
<div id="universe-set-of-genes" class="section level2 hasAnchor" number="1.3">
<h2><span class="header-section-number">1.3</span> Universe set of genes<a href="index.html#universe-set-of-genes" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p><strong>all genes having quantified cassette exons</strong></p>
<div id="go-ensembl" class="section level3 hasAnchor" number="1.3.1">
<h3><span class="header-section-number">1.3.1</span> GO (ENSEMBL)<a href="index.html#go-ensembl" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb3-1"><a href="index.html#cb3-1" aria-hidden="true" tabindex="-1"></a><span class="co"># read table with cassette exons and select the genes</span></span>
<span id="cb3-2"><a href="index.html#cb3-2" aria-hidden="true" tabindex="-1"></a>universe <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">&quot;data/cassette_exons_universe_table.tsv&quot;</span>, </span>
<span id="cb3-3"><a href="index.html#cb3-3" aria-hidden="true" tabindex="-1"></a>                       <span class="at">header =</span> T) <span class="sc">%&gt;%</span> dplyr<span class="sc">::</span><span class="fu">select</span>(gene_id) <span class="sc">%&gt;%</span> <span class="fu">unlist</span>()</span>
<span id="cb3-4"><a href="index.html#cb3-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-5"><a href="index.html#cb3-5" aria-hidden="true" tabindex="-1"></a><span class="co"># select unique IDs</span></span>
<span id="cb3-6"><a href="index.html#cb3-6" aria-hidden="true" tabindex="-1"></a>universe <span class="ot">&lt;-</span> <span class="fu">unique</span>(<span class="fu">unlist</span>(<span class="fu">strsplit</span>(universe, <span class="st">&quot;,&quot;</span>)))</span>
<span id="cb3-7"><a href="index.html#cb3-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-8"><a href="index.html#cb3-8" aria-hidden="true" tabindex="-1"></a><span class="co"># preview first 10 entrees</span></span>
<span id="cb3-9"><a href="index.html#cb3-9" aria-hidden="true" tabindex="-1"></a><span class="fu">head</span>(universe, <span class="dv">10</span>)</span></code></pre></div>
<pre><code>##  [1] &quot;ENSG00000174371&quot; &quot;ENSG00000117009&quot; &quot;ENSG00000284770&quot; &quot;ENSG00000285053&quot; &quot;ENSG00000143774&quot; &quot;ENSG00000143801&quot; &quot;ENSG00000143353&quot;
##  [8] &quot;ENSG00000117691&quot; &quot;ENSG00000117625&quot; &quot;ENSG00000143469&quot;</code></pre>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb5-1"><a href="index.html#cb5-1" aria-hidden="true" tabindex="-1"></a><span class="co"># length of a vector with ENSEMBL IDs</span></span>
<span id="cb5-2"><a href="index.html#cb5-2" aria-hidden="true" tabindex="-1"></a><span class="fu">length</span>(universe)  </span></code></pre></div>
<pre><code>## [1] 10273</code></pre>
</div>
<div id="kegg-entrez" class="section level3 hasAnchor" number="1.3.2">
<h3><span class="header-section-number">1.3.2</span> KEGG (ENTREZ)<a href="index.html#kegg-entrez" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb7-1"><a href="index.html#cb7-1" aria-hidden="true" tabindex="-1"></a><span class="co"># convert ENSEMBL IDs into ENTREZ format (required for KEGG analysis)</span></span>
<span id="cb7-2"><a href="index.html#cb7-2" aria-hidden="true" tabindex="-1"></a>ENTREZ_universe <span class="ot">&lt;-</span> <span class="fu">bitr</span>(universe, <span class="at">fromType =</span> <span class="st">&quot;ENSEMBL&quot;</span>, </span>
<span id="cb7-3"><a href="index.html#cb7-3" aria-hidden="true" tabindex="-1"></a>                        <span class="at">toType =</span> <span class="st">&quot;ENTREZID&quot;</span>, <span class="at">OrgDb=</span> organismDB) <span class="sc">%&gt;%</span> </span>
<span id="cb7-4"><a href="index.html#cb7-4" aria-hidden="true" tabindex="-1"></a>                        dplyr<span class="sc">::</span><span class="fu">select</span>(ENTREZID) <span class="sc">%&gt;%</span> <span class="fu">unlist</span>()</span></code></pre></div>
<pre><code>## &#39;select()&#39; returned 1:many mapping between keys and columns</code></pre>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb9-1"><a href="index.html#cb9-1" aria-hidden="true" tabindex="-1"></a><span class="co"># select unique IDs</span></span>
<span id="cb9-2"><a href="index.html#cb9-2" aria-hidden="true" tabindex="-1"></a>ENTREZ_universe <span class="ot">&lt;-</span> <span class="fu">unique</span>(<span class="fu">unlist</span>(<span class="fu">strsplit</span>(ENTREZ_universe, <span class="st">&quot;,&quot;</span>)))</span>
<span id="cb9-3"><a href="index.html#cb9-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-4"><a href="index.html#cb9-4" aria-hidden="true" tabindex="-1"></a><span class="co"># preview first 10 entrees</span></span>
<span id="cb9-5"><a href="index.html#cb9-5" aria-hidden="true" tabindex="-1"></a><span class="fu">head</span>(ENTREZ_universe)</span></code></pre></div>
<pre><code>## [1] &quot;9156&quot;   &quot;8564&quot;   &quot;6905&quot;   &quot;2987&quot;   &quot;5664&quot;   &quot;127018&quot;</code></pre>
<div class="sourceCode" id="cb11"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb11-1"><a href="index.html#cb11-1" aria-hidden="true" tabindex="-1"></a><span class="co"># length of a vector with ENSEMBL IDs</span></span>
<span id="cb11-2"><a href="index.html#cb11-2" aria-hidden="true" tabindex="-1"></a><span class="fu">length</span>(ENTREZ_universe)  </span></code></pre></div>
<pre><code>## [1] 10220</code></pre>
</div>
</div>
<div id="render-the-book" class="section level2 hasAnchor" number="1.4">
<h2><span class="header-section-number">1.4</span> Render the book<a href="index.html#render-the-book" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb13-1"><a href="index.html#cb13-1" aria-hidden="true" tabindex="-1"></a><span class="co"># the full book is rendered with this command</span></span>
<span id="cb13-2"><a href="index.html#cb13-2" aria-hidden="true" tabindex="-1"></a>bookdown<span class="sc">::</span><span class="fu">render_book</span>(<span class="st">&quot;index.Rmd&quot;</span>, <span class="st">&quot;bookdown::gitbook&quot;</span>)</span></code></pre></div>
</div>
<div id="session-info" class="section level2 hasAnchor" number="1.5">
<h2><span class="header-section-number">1.5</span> Session info<a href="index.html#session-info" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div class="sourceCode" id="cb14"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb14-1"><a href="index.html#cb14-1" aria-hidden="true" tabindex="-1"></a><span class="fu">sessionInfo</span>()</span></code></pre></div>
<pre><code>## R version 4.2.2 (2022-10-31)
## Platform: x86_64-apple-darwin17.0 (64-bit)
## Running under: macOS Monterey 12.0.1
## 
## Matrix products: default
## LAPACK: /Library/Frameworks/R.framework/Versions/4.2/Resources/lib/libRlapack.dylib
## 
## locale:
## [1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8
## 
## attached base packages:
## [1] stats4    stats     graphics  grDevices utils     datasets  methods   base     
## 
## other attached packages:
##  [1] matrixTests_0.1.9.1   DT_0.25               org.Hs.eg.db_3.15.0   AnnotationDbi_1.58.0  IRanges_2.30.1        S4Vectors_0.34.0     
##  [7] Biobase_2.56.0        BiocGenerics_0.42.0   enrichplot_1.16.2     pathview_1.36.1       clusterProfiler_4.4.4 forcats_0.5.2        
## [13] stringr_1.5.0         dplyr_1.0.10          purrr_0.3.4           readr_2.1.2           tidyr_1.2.1           tibble_3.1.8         
## [19] ggplot2_3.4.0         tidyverse_1.3.2      
## 
## loaded via a namespace (and not attached):
##   [1] readxl_1.4.1           shadowtext_0.1.2       backports_1.4.1        fastmatch_1.1-3        plyr_1.8.8             igraph_1.3.5          
##   [7] lazyeval_0.2.2         splines_4.2.2          crosstalk_1.2.0        BiocParallel_1.30.3    GenomeInfoDb_1.32.4    digest_0.6.29         
##  [13] yulab.utils_0.0.5      htmltools_0.5.3        GOSemSim_2.22.0        viridis_0.6.2          GO.db_3.15.0           fansi_1.0.4           
##  [19] magrittr_2.0.3         memoise_2.0.1          googlesheets4_1.0.1    tzdb_0.3.0             Biostrings_2.64.1      graphlayouts_0.8.1    
##  [25] modelr_0.1.9           matrixStats_0.63.0     colorspace_2.1-0       blob_1.2.3             rvest_1.0.3            ggrepel_0.9.2         
##  [31] haven_2.5.1            xfun_0.33              crayon_1.5.1           RCurl_1.98-1.8         jsonlite_1.8.0         graph_1.74.0          
##  [37] scatterpie_0.1.8       ape_5.6-2              glue_1.6.2             polyclip_1.10-0        gtable_0.3.1           gargle_1.2.1          
##  [43] zlibbioc_1.42.0        XVector_0.36.0         Rgraphviz_2.40.0       scales_1.2.1           DOSE_3.22.1            DBI_1.1.3             
##  [49] Rcpp_1.0.10            viridisLite_0.4.1      gridGraphics_0.5-1     tidytree_0.4.1         bit_4.0.4              htmlwidgets_1.5.4     
##  [55] httr_1.4.4             fgsea_1.22.0           RColorBrewer_1.1-3     ellipsis_0.3.2         pkgconfig_2.0.3        XML_3.99-0.10         
##  [61] farver_2.1.1           sass_0.4.2             dbplyr_2.2.1           utf8_1.2.2             labeling_0.4.2         ggplotify_0.1.0       
##  [67] tidyselect_1.1.2       rlang_1.0.6            reshape2_1.4.4         munsell_0.5.0          cellranger_1.1.0       tools_4.2.2           
##  [73] cachem_1.0.6           downloader_0.4         cli_3.6.0              generics_0.1.3         RSQLite_2.2.17         broom_1.0.1           
##  [79] evaluate_0.16          fastmap_1.1.0          yaml_2.3.5             ggtree_3.4.2           knitr_1.40             bit64_4.0.5           
##  [85] fs_1.5.2               tidygraph_1.2.2        KEGGREST_1.36.3        ggraph_2.0.6           nlme_3.1-160           KEGGgraph_1.56.0      
##  [91] aplot_0.1.7            DO.db_2.9              xml2_1.3.3             compiler_4.2.2         rstudioapi_0.14        png_0.1-7             
##  [97] reprex_2.0.2           treeio_1.20.2          tweenr_2.0.2           bslib_0.4.0            stringi_1.7.12         highr_0.9             
## [103] lattice_0.20-45        Matrix_1.5-1           vctrs_0.5.2            pillar_1.8.1           lifecycle_1.0.3        jquerylib_0.1.4       
## [109] data.table_1.14.6      bitops_1.0-7           patchwork_1.1.2        qvalue_2.28.0          R6_2.5.1               bookdown_0.29         
## [115] gridExtra_2.3          codetools_0.2-18       MASS_7.3-58.1          assertthat_0.2.1       withr_2.5.0            GenomeInfoDbData_1.2.8
## [121] parallel_4.2.2         hms_1.1.2              grid_4.2.2             ggfun_0.0.7            rmarkdown_2.16         googledrive_2.0.0     
## [127] ggforce_0.3.4          lubridate_1.8.0</code></pre>

</div>
</div>
            </section>

          </div>
        </div>
      </div>

<a href="go-and-kegg-common-targets-06.html" class="navigation navigation-next navigation-unique" aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/rstudio/bookdown-demo/edit/master/index.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["GO_and_KEGG_analysis.pdf", "GO_and_KEGG_analysis.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
